### PR TITLE
Added 500 error to Fastly Custom Error Page list

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request

This pull request (PR) adds a mention of 500 error - it is not clear on the documentation that 500 is covered.
This came from someone seeking help on #cloud 

## Affected DevDocs pages

https://devdocs.magento.com/cloud/cdn/cloud-fastly-custom-response.html

### CODE REVIEW

File: src/cloud/cdn/cloud-fastly-custom-response.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Currently, you can customize the following Fastly response pages for your {{ sit
 
 -  [Timeout or site maintenance outages (503 Service Unavailable)](#customize-the-503-error-page)
 -  [WAF blocking events that occur when the WAF detects suspicious request traffic (403 Forbidden)](#customize-the-waf-error-page)
+-  [Server Error (500 Internal Server Error)](#customize-the-503-error-page) - this is covered by the same function as the 503 error.
```
Comment (meker12) [inline src/cloud/cdn/cloud-fastly-custom-response.md:24]:
```suggestion
-  [Server errors - Internal server error, timeout or site maintenance outages (error code 500 or greater)](#customize-the-503-error-page)
-  [WAF blocking events that occur when the WAF detects suspicious request traffic (403 Forbidden)](#customize-the-waf-error-page)
```

**Review by meker12 — CHANGES_REQUESTED** [review]

File: src/cloud/cdn/cloud-fastly-custom-response.md (hunk 1 above)
Comment (BarnyShergold) [reply src/cloud/cdn/cloud-fastly-custom-response.md:24]: We can go with that. Main reason I added separately was because of a complaint about clarity.

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@meker12 the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-9096`.

**Pull request merged**